lib/benchmarks.nix

```diff
@@ -196,17 +196,19 @@ in rec {
   */
   mkBenchLWAFTR = { snabb
                   , times
-                  , duration
-                  , mode
-                  , ipv4PCap
-                  , ipv6PCap
-                  , conf
+                  , duration ? "10"
+                  , mode ? "bare"
+                  , ipv4PCap ? "ipv4-0550.pcap"
+                  , ipv6PCap ? "ipv6-0550.pcap"
+                  , conf ? "icmp_on_fail.conf"
+                  , loadTestStep ? "0.2e9"
```

Comment:
Defaults are useful it you're not using the same jobset, which may happen in future. Currently it's no harm since values are passed in any case. I'm fine removing them, but for other benchmarks we ended up providing defaults for benchmarks themselves.